contracts/rounds/RoundsManager.sol

```diff
@@ -163,6 +204,10 @@ contract RoundsManager is ManagerProxyTarget, IRoundsManager {
         return blockNum().sub(currentRoundStartBlock()) >= roundLength.sub(lockedBlocks);
     }
 
+    function currentRoundIsRewardRound() public view returns (bool) {
+        return nextRewardRound.sub(1) == currentRound();
```

Comment:
Shouldn't this be `nextRewardRound == currentRound` ?  Otherwise L134 would have to be `if (currRound >= nextRewardRound` , no?